website/content/docs/commands/fix.mdx

```diff
@@ -10,6 +10,8 @@ sidebar_title: <tt>fix</tt>
 
 # `fix` Command
 
+-> **Note** This command is not available on HCL2 template yet, it will be added when we need to introduce the first 'fix'.
```

Comment:
```suggestion
-> **Note** This command is not available on HCL2 templates yet, it will be added when we need to introduce the first 'fix'.
```

Sorry, forgot the S !